src/components/Upload/Upload.tsx

```diff
@@ -554,7 +554,11 @@ const InternalUpload: React.ForwardRefRenderFunction<unknown, UploadProps> = (
         classNames={styles.uploadDropButton}
         disabled={mergedDisabled}
         htmlType={'button'}
-        onKeyDown={(event) => event.preventDefault()}
+        onKeyDown={(event: React.KeyboardEvent<HTMLButtonElement>) => {
+          if (event.key !== eventKeys.TAB) {
```

Comment:
does this appropriate support Shift+TAB as well?

Reply:
Will update to account for shift. Same will work for the local button.

Reply:
Done